build/karma.conf.babel.js

```diff
@@ -1,9 +1,9 @@
-import { argv } from 'yargs'
-
-import config from '../config'
-import webpackConfig from './webpack.config'
+const yargs = require('yargs')
+const config = require('../config')
+const webpackConfig = require('./webpack.config')
 
 const { paths } = config
+const { argv } = yargs
```

Comment:
Does the style guide say anything about just doing `const argv = require('yargs').argv`, actually that's silly, I am pretty sure you can do this:
``` js
const { argv } = require('yargs')
```

Yeah I am pretty sure that's doable.